packages/build-tools/src/utils/prebuild.ts

```diff
@@ -0,0 +1,49 @@
+import { Job } from '@expo/eas-build-job';
+import spawnAsync from '@expo/turtle-spawn';
+
+import { BuildContext } from '../context';
+
+export interface PrebuildOptions {
+  extraEnvs?: Record<string, string>;
+}
+
+export async function prebuildAsync<TJob extends Job>(
+  ctx: BuildContext<TJob>,
+  options?: PrebuildOptions
+): Promise<void> {
+  const spawnOptions = {
```

Comment:
```suggestion
  const spawnOptions: SpawnOptions = {
```
+ import `SpawnOptions`